dev/com.ibm.ws.microprofile.openapi_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -26,6 +26,8 @@
     </properties>
 
     <build>
+        <directory>${targetDirectory}</directory>
+        <defaultGoal>clean test</defaultGoal>
```

Comment:
These lines are in all of the other TCKs.

It's necessary to add at least the `<directory>` line to ensure that the results end up in the same location as all other TCKs so that we can use common code to extract and publish the results.